lib/analyzer/os-release/docker.ts

```diff
@@ -76,5 +76,14 @@ export async function detect(
     osRelease.name = "oracle";
   }
 
+  // Support round version. ie change SLES 15 to SLES 15.0
+  if (
+    osRelease.name.trim() === "sles" &&
+    osRelease.version &&
+    !osRelease.version.includes(".")
+  ) {
+    osRelease.version += ".0";
+  }
+
```

Comment:
I don't like it but I think this whole function needs a cleaner re-write, happy to leave it as is right now